tests/playwright/sanity/container.test.js

```diff
@@ -526,6 +528,30 @@ test.describe( 'Container tests', () => {
 		await editor.togglePreviewMode();
 		await expect( resizers ).toHaveCount( 0 );
 	} );
+
+	test( 'Test grid container controls', async ( { page }, testInfo ) => {
+		// Arrange.
+		const wpAdmin = new WpAdminPage( page, testInfo );
+		const editor = await wpAdmin.useElementorCleanPost();
+
+		// Add container.
+		await editor.addElement( { elType: 'container' }, 'document' );
+
+		// Close Navigator
+		await editor.closeNavigatorIfOpen();
+
+		// Set container type to grid.
+		await page.locator( '.elementor-control-container_type select' ).selectOption( 'grid' );
+
+		// Set various controls
+		await page.locator( '.elementor-control-grid_justify_items .eicon-align-center-v' ).click();
```

Comment:
why not check all options ?
what about front end ?

Reply:
Done.
I've done checks for all the settings, in the editor and frontend.